sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -433,13 +437,13 @@ export class ApplicationStudentsController extends BaseController {
   })
   @Get(":applicationNumber/appeal")
   async getApplicationToRequestAppeal(
-    @Param("applicationNumber") applicationNumber: string,
+    @Param() applicationNumberParam: ApplicationNumberParamAPIInDTO,
```

Comment:
so, will this be an object. and what about `applicationNumber` in line 438.

Reply:
It will be still a param only but represented in an object, like the way how pagination query params are done now. 
The advantage of putting them into object is, we will get the OOTB class validator features instead of we having to create one.

Reply:
good to know